GeneratorInterface/Core/interface/TaggedProtonHepMCFilter.h

```diff
@@ -0,0 +1,24 @@
+#ifndef __TAGGEDPROTONHEPMCFILTER__
+#define __TAGGEDPROTONHEPMCFILTER__
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include "GeneratorInterface/Core/interface/BaseHepMCFilter.h"
+
+class TaggedProtonHepMCFilter : public BaseHepMCFilter {
+private:
+  const int proton_PDGID_ = 2212;
+  const int neutron_PDGID_ = 2112;
+  const double OneOverbeamEnergy_;
```

Comment:
Since you have to apply the code-checks anyhow, please also fix this one that I forgot
```suggestion
  const double oneOverbeamEnergy_;
```
(class members should start with lowercase letter, according to the CMS standards)

Reply:
thanks, I fixed the variable in the source and the header in [30964dc](https://github.com/cms-sw/cmssw/pull/40083/commits/30964dcc7553b2d15656c2dc2b522d41cf369210) and [
f6d6fa7](https://github.com/cms-sw/cmssw/pull/40083/commits/f6d6fa7f94cc044c0b779293e0c4d2792dcf79fc)